integration_tests/run_tests.yaml

```diff
@@ -3,7 +3,6 @@
 steps:
 - name: gcr.io/gcp-runtimes/integration_test
   args:
-    - '--no-deploy'
     - '--url'
     - 'https://${_VERSION_ID}-dot-${PROJECT_ID}.appspot.com'
```

Comment:
just to be sure, where does your application get deployed? are you sure you don't want to just let the framework do the deploy for you?